src/cdomains/congruenceClosure.ml

```diff
@@ -127,15 +127,15 @@ module Disequalities = struct
   let is_empty = TMap.is_empty
   let remove = TMap.remove
   (** Returns a list of tuples, which each represent a disequality *)
-  let bindings =
-    List.flatten %
+  let bindings x =
+    List.flatten @@
     List.concat_map
       (fun (t, smap) ->
          List.map (fun (z, tset) ->
              List.map (fun term ->
                  (t,z,term)) (TSet.elements tset))
            (ZMap.bindings smap)
-      ) % TMap.bindings
+      ) @@ TMap.bindings x
```

Comment:
Given that we're already touching this code, this seems like a place where `Seq` should really be used instead of all the `List` business: this is basically a nesting of 3 `map`s and 2 `flatten`s (with `concat_map` being both).

Reply:
Done!